client/src/components/FeedPost/index.tsx

```diff
@@ -3,64 +3,143 @@ import '../../styles/style.scss';
 import '../Post/style.scss';
 import noAvatar from '../../assets/noAvatar.svg';
 import {Link} from 'react-router-dom';
+import TextareaAutosize from 'react-textarea-autosize';
+import { formatDescription } from '../../helpers/regex.description';
+import { IUserData } from '../Profile';
 
-export default class FeedPost extends React.Component<any> {
+interface IBody {
+    userId: string;
+    postId: string;
+}
+
+interface IProps {
+    id: any;
+    newsFeed: any;
+    author: any;
+    image: any;
+    description: string;
+    authorsOfLike: any;
+    user: IUserData;
+    getNewsFeedAsync: () => void;
+    getMoreNewsFeedAsync: (page: number) => void;
+    addLoggedUserLike: (loggedUserId: string, postId: string, authorsOfLike: []) => void;
+    removeLoggedUserLike: (loggedUserId: string, postId: string, authorsOfLike: []) => void;
+    addLike: (body: IBody) => void;
+    setCountOfLikes: (countOfLikes: number) => void;
+    deleteLike: (body: IBody) => void;
+    countOfLikes: number;
+    likeExist: boolean;
```

Comment:
do we need this props if we have authorsOfLike.length ?

Reply:
We have only count of likes from server but we need on our feed to add or delete to post or from post, and we need this for adding or removing(without countOfLikes, this is unnecessary)